editoast/src/models/scenario.rs

```diff
@@ -42,6 +41,20 @@ pub struct Scenario {
     pub electrical_profile_set_id: Option<i64>,
 }
 
+#[derive(thiserror::Error, Debug)]
+pub enum Error {
+    #[error("Scenario with id {scenario_id} not found")]
+    NotFound { scenario_id: i64 },
+    #[error(transparent)]
+    Database(#[from] editoast_models::Error),
+}
+
+impl From<database::DatabaseError> for Error {
+    fn from(e: database::DatabaseError) -> Self {
+        Error::Database(e.into())
+    }
+}
```

Comment:
```suggestion
    #[error(transparent)]
    #[from(forward)]
    Database(editoast_models::Error),
}
```

a nice trick, not always applicable depending on how the error is used